tests/quantization/test_range_init.py

```diff
@@ -455,6 +452,8 @@ def __len__(self):
             "input_info": {
                 "sample_size": [1, 3, 100, 100]
             },
+            "quantizer_setup_type": 'pattern_based',
+            "target_device": "NONE",
```

Comment:
These two definitions should be mutually exclusive.

Reply:
Fixed.